src/bantz/data/graph_store.py

```
@@ -0,0 +1,235 @@
+
+    # ── decay & reinforcement ────────────────────────────────────
+
+    async def apply_decay(
+        self,
+        edge_id: str,
+        decay_rate: float = 0.05,
+        reference_time: Optional[float] = None,
+    ) -> float:
+        """Reduce edge weight based on elapsed time.
+
+        ``new_weight = weight × (1 - decay_rate) ^ days_elapsed``
```

Ruff (0.15.0)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)